src/blight/cli.py

```diff
@@ -26,14 +27,35 @@ def _export_guess_wrapped():
         _export(f"BLIGHT_WRAPPED_{variable}", tool_path)
 
 
+def _swizzle_path():
+    blight_dir = f"/tmp/blight-XXX"
```

Comment:
Let's use `tempfile.mkdtemp` here: https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp

I.e., every time the user runs `--swizzle-path`, it should create a new temp dir. This is a little wasteful, but conceptually cleaner (and in normal usage they should only be running it once, anyways).